components/gif_picker/static/icons/ic_reactions_blue.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are we using the blue and grey versions of this image? Can we remove them if we're not?